mathics/builtin/atomic/strings.py

```diff
@@ -739,8 +739,8 @@ def apply(self, s, n, expression, evaluation):
 
     def apply_truncated(self, s, n, m, expression, evaluation):
         "StringRepeat[s_String, n_Integer, m_Integer]"
-        py_n = n.get_int_value() if isinstance(n, Integer) else 0
-        py_m = m.get_int_value() if isinstance(m, Integer) else 0
+        py_n = n.value if isinstance(n, Integer) else 0
```

Comment:
m and n must be integers, according to the rule.

Reply:
Good point.